.travis.yml

```diff
@@ -1,11 +1,14 @@
 language: python
 env:
   - DJANGO_SETTINGS_MODULE=concordia.settings_test
-python:
-  - "3.6"
-  - "3.6-dev"  # 3.6 development branch
-  - "3.7"
-  - "3.7-dev"  # 3.7 development branch
+matrix:
+  include:
+    - python: "3.6"
+    - python: "3.6-dev"
+    - python: "3.7-dev"
+    - python: "3.7"
+      dist: xenial
+      sudo: true
```

Comment:
You need the same config for 3.7-dev, otherwise 3-7-dev is just an old beta of 3.7.0 and not a pre release of the next minor version.

Reply:
Oh, that's quite a footnote: https://docs.travis-ci.com/user/languages/python/#development-releases-support

I'm guessing that means we can go back to a simple `python:` block because we should use xenial everywhere.